MANIFEST.in

```diff
@@ -1,4 +1,8 @@
 recursive-include panoramix/templates *
 recursive-include panoramix/static *
+recursive-exclude panoramix/static/assets/node_modules *
```

Comment:
This excludes the thousands of files in that folder to ship with the python package